### PR TITLE
Update bigquery-connector-example

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL. The WSO2 EI BigQuery connector uses user credentials to ensure that the application has access only to BigQuery tables that are available to the end user.
+
+The WSO2 EI BigQuery connector includes data retrieving operations to retrieve BigQuery data such as project details, datasets, tables and jobs. Our connector allows only one operation to insert data into the BigQuery tables. During this operation the user able to inserts data into the created table in BigQuery. In this example we are trying to build up a sample scenario based on the BigQuery Table operations.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:5]: The WSO2 EI BigQuery connector is mostly comprised of operations which is useful for retrieving BigQuery data such as project details, datasets, tables and jobs (it has one operation which can be used to insert data into BigQuery tables). 

In this example we are trying to build up a sample scenario based on the BigQuery Table operations.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # BigQuery Connector Example
 
-The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL. The WSO2 EI BigQuery connector uses user credentials to ensure that the application has access only to BigQuery tables that are available to the end user.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:3]: The WSO2 EI BigQuery connector uses user credentials to ensure that the application has access only to BigQuery tables that are available to the end user.

- it is not clear what is meant by this? Could you explain more?

Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:3]: Removed this section.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:5]: Updated.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
 
+The WSO2 EI BigQuery connector is mostly comprised of operations which is useful for retrieving BigQuery data such as project details, datasets, tables and jobs (it has one operation which can be used to insert data into BigQuery tables).
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:5]:
```suggestion
The WSO2 EI BigQuery connector is mostly comprised of operations that are useful for retrieving BigQuery data such as project details, datasets, tables, and jobs (it has one operation that can be used to insert data into BigQuery tables).
```

Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:5]: Updated.

**Pull request merged**